doc/lsst.gauss2d.fit/index.rst

```diff
@@ -0,0 +1,58 @@
+.. py:currentmodule:: lsst.gauss2d.fit
```

Comment:
I hadn't realized the Python package was going to be `lsst.gauss2d.fit` rather than `lsst.gauss2dfit`.  Does this actually work with `lsst.gauss2d` being provided by another git/eups package?  In other cases where we have a Python package split across multiple git/eups packages, we have to put them at the same level, since there can only be one (e.g.) `lsst/gauss2d/__init__.py` (either the `pkgutils` one that splits sibling packages, or a regular one).

There's some chance that this could work if `PYTHONPATH` has the two package directories in one order, and not if they are in the other order.